test/e2e/framework/volume/fixtures.go

```diff
@@ -543,50 +543,48 @@ func GenerateScriptCmd(command string) []string {
 	return commands
 }
 
-// generateWriteBlockCmd generates the corresponding command lines to write to a block device the given content.
-// Depending on the Node OS is Windows or linux, the command will use powershell or /bin/sh
-func generateWriteBlockCmd(content, fullPath string) []string {
+func generateWriteCmd(content, path string) []string {
 	var commands []string
 	if !framework.NodeOSDistroIs("windows") {
-		commands = []string{"/bin/sh", "-c", "echo '" + content + "' > " + fullPath}
+		commands = []string{"/bin/sh", "-c", "echo '" + content + "' > " + path}
 	} else {
-		commands = []string{"powershell", "/c", "echo '" + content + "' > " + fullPath}
+		commands = []string{"powershell", "/c", "echo '" + content + "' > " + path}
 	}
 	return commands
 }
 
-// generateWriteFileCmd generates the corresponding command lines to write a file with the given content and file path.
```

Comment:
Keep generateReadBlockCmd and generateWriteBlockCmd, generateWriteFileCmd and generateReadFileCmd closer.

Reply:
common function generateWriteCmd  for generateWriteBlockCmd and generateWriteFileCmd